infra/monitoring/functions/metric_pusher/f_metric_pusher/main.py

```diff
@@ -1,47 +1,58 @@
 from f_metric_pusher.service import cloudwatch_service, athena_service
-from f_metric_pusher.queries import COUNT_QUERY, LATENCY_QUERY
-from f_metric_pusher.env_variables import db_table_names, output_format
+from f_metric_pusher.queries import COUNT_QUERY, LATENCY_HUDI_QUERY, LATENCY_JSON_QUERY
+from f_metric_pusher.env_variables import database_name, table_name, output_format
+from datetime import datetime
 
 
-def count_query_push_metric(metric_name, db_table_names):
+def count_query_push_metric(metric_name, database_name, table_name):
+
     count_query_result = athena_service.execute_query(
         COUNT_QUERY.format(
-            database_name=db_table_names[output_format][0],
-            table_name=db_table_names[output_format][1],
+            database_name=database_name,
+            table_name=table_name,
         )
-    )
-
+    )[1]["Data"][0]["VarCharValue"]
+    
     cloudwatch_service.push_metric(
         metric={
             "metric_name": metric_name,
-            "table_name": db_table_names[output_format][1],
-            "value": count_query_result,
-        }
+            "table_name": table_name,
+            "value": int(count_query_result),
+            "timestamp": datetime.now(),
+        },
     )
 
-
-def latency_query_push_metric(metric_name, db_table_names):
-    count_query_result = athena_service.execute_query(
-        LATENCY_QUERY.format(
-            database_name=db_table_names[output_format][0],
-            table_name=db_table_names[output_format][1],
+def latency_query_push_metric(metric_name, database_name, table_name):
+
+    if output_format == "hudi":
+        latency_query_imported = LATENCY_HUDI_QUERY
+    else:
```

Comment:
Be specific so this can be easily extensible, instead of `else`  it is better to do do `if output_format == "json"`